aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role/samples/sample9.cs

```diff
@@ -2,16 +2,16 @@
 using Owin;
 using WebRole1.PersistentConnections;
 
-[assembly: OwinStartupAttribute(typeof(WebRole1.Startup))]
+[assembly: OwinStartup(typeof(WebRole1.Startup))]
 namespace WebRole1
 {
     public partial class Startup
     {
         public void Configuration(IAppBuilder app)
         {
-            ConfigureAuth(app);     //Only needed if "No Authentication" was not selected for the project
+            ConfigureAuth(app); // Only needed if "No Authentication" was not selected for the project
```

Comment:
newline after `ConfigureAuth(app); // Only needed if "No Authentication" `
so you don't get any horizontal scroll bars. The rule is, create the widest brower window with no left or right TOC - then make sure your code doesn't require horizontal scroll bars